rust/common-macros/tests/new_type.rs

```diff
@@ -6,8 +6,8 @@ fn it_derives_from_inner() {
     struct Foo(pub String);
 
     let s = String::from("foo");
-    let foo = Foo::from(s.clone());
-    assert_eq!(foo.0, s);
```

Comment:
I don't know why this wasn't caught before, but clippy doesn't allow names like 'foo'.

https://rust-lang.github.io/rust-clippy/master/index.html#disallowed_names